country-currency-map.js

```diff
@@ -26,7 +27,20 @@ function formatCurrency(value, currencyAbbr) {
     return `${value} ${currencyAbbr}`;
 }
 
+function getCurrencyList() {
+    return currencyMap;
+}

```

Comment:
Maybe I'm not understanding how this will be used for the currency dropdown. Whatever you think is best, let's go with that 😄